build-tests/x86/tumbleweed/test-image-wsl-ng/root/etc/wsl-distribution.conf

```diff
@@ -0,0 +1,3 @@
+[oobe]
+defaultUid = 1000
+defaultName = SUSE
```

Comment:
I believe you'll need to specify a [setup script to run on first use](https://src.fedoraproject.org/rpms/wsl-setup/blame/wsl-distribution.conf?identifier=rawhide#_13), like [this one](https://src.fedoraproject.org/rpms/wsl-setup/blob/rawhide/f/wsl-oobe.sh).